features/attached_enable.feature

```diff
@@ -266,3 +266,30 @@ Feature: Enable command behaviour when attached to an UA subscription
             FIPS enabled
             A reboot is required to complete install
             """
+
+    @series.bionic
+    @series.xenial
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached enable livepatch on a machine with fips active
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `canonical-livepatch status` with sudo
+        Then I will see the following on stderr:
```

Comment:
Let's add a When/Then for running ua status and validating that stdout regexp matches fips +enabled too